File: CP Lab/Lab4/8.c
// Lab project by Sarang Dev Saha (20UCS173)
//LAB4_QUES 8
// Take any number as user input and print whether it is an even number or odd.
#include <stdio.h>
int main() {
    int num;
    printf("Enter an integer: ");
    scanf("%d", &num);

    // If num is perfectly divisible by 2
    if(num % 2 == 0)
        printf("The number is EVEN.", num);
    else
        printf("The number is ODD.", num);
    
    return 0;
}

File: CP Lab/Lab2/8.py
# Lab Project by Sarang Dev Saha

# LAB_02 QUE_08
# Take 3 numbers from the user and print the second largest and largest number without using any iterative statements and inbuilt functions. 

# Taking 3 numbers from user
a=float(input("Enter the first number:"))
b=float(input("Enter the second number:"))
c=float(input("Enter the third number:"))
list=[a, b, c]
# Using sort() function to arrange numbers in ascending order
list. sort() 
# Printing Maximum number
print("Maximum number is",list[2])
# Printing second maximum number
print("Second Maximum number is",list[1])